my_digital_being/activities/activity_post_recent_memory_tweet.py

```diff
@@ -168,6 +169,29 @@ def _get_memories_used_last_time(self, shared_data) -> List[str]:
                     return used
         return []
 
+    def _sync_activity_config(self):
+        """
+        synchronize activity config
+        """
+        # Load the config from the being
+        from framework.main import DigitalBeing
+        being = DigitalBeing()
+        being.initialize()
+        system_prompt = (
+            f"Please craft a short tweet (under {self.max_length} chars) that references these memories, "
+            f"reflects the personality and objectives, and ensures it's not repetitive or dull. "
+            f"Keep it interesting, cohesive, and mindful of the overall tone.\n"
+        )
+        num_activities_to_fetch = 10
+        if "activity_constraints" in being.configs and \
+            "activities_config" in being.configs["activity_constraints"] and \
+            "PostRecentMemoriesTweetActivity" in being.configs["activity_constraints"]["activities_config"]:
+            activities_config = being.configs["activity_constraints"]["activities_config"]["PostRecentMemoriesTweetActivity"]
+            system_prompt = activities_config.get("system_prompt", system_prompt)
+            num_activities_to_fetch = activities_config.get("num_activities_to_fetch", num_activities_to_fetch)
+        self.system_prompt = system_prompt
+        self.num_activities_to_fetch = num_activities_to_fetch
```

Comment:
Seems like this duplicates the logic from the same function in post_tweet, and it this brought up another question of how we should cleanly manage customized config vs default config for each skill. imo we don't need to solve it here but something interesting to think about. 

@yoheinakajima what do you think?

Reply:
looks like https://github.com/pippinlovesyou/pippin/pull/29/files addressed some of this